### PR TITLE
December22 new app list

### PR DESCRIPTION
**Pull request by shuvashish76**
- ~OpenCamera Sensors~ - archived
- Little File Explorer
- Karma Firewall
- Mastodon - updated fdroid link
- Megalodon
- Jami
- Meshenger
- GuitarFretboardTrainer
- InnerTune
- ~Quillnote~ replaced with it's fork Quillpad (https://github.com/msoultanidis/quillnote/issues/177#issuecomment-1319644880)
- Aggregator
- ChemistryLab
- Inure App Manager (https://gitlab.com/fdroid/rfp/-/issues/2247)
- Permission Pilot (https://gitlab.com/fdroid/rfp/-/issues/2260)
- Captive Portal Controller (https://gitlab.com/fdroid/rfp/-/issues/2233)
- Sensorkaren
- Sudoku

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -556,6 +564,9 @@ A list of **Free** and **Open Source Software** ***(FOSS)*** for **Android** –
 * [**SuperFreezZ**](https://gitlab.com/SuperFreezZ/SuperFreezZ) <sup>**[[F-Droid](https://f-droid.org/app/superfreeze.tool.android)]**</sup>
 * [**Battery Tool**](https://github.com/Domi04151309/BatteryTool) <sup>**[[F-Droid](https://f-droid.org/app/io.github.domi04151309.batterytool)]**</sup>
 * [**AppManager**](https://github.com/MuntashirAkon/AppManager) <sup>**[[F-Droid](https://f-droid.org/app/io.github.muntashirakon.AppManager)]**</sup>
+* [**Inure App Manager**](https://github.com/Hamza417/Inure)
+* [**Permission Pilot**](https://github.com/d4rken-org/permission-pilot)
```
Comment (offa) [inline README.md:568]: F-Droid discussion: https://github.com/d4rken-org/permission-pilot/issues/148

File: README.md [hunk 2]
```diff
@@ -556,6 +564,9 @@ A list of **Free** and **Open Source Software** ***(FOSS)*** for **Android** –
 * [**SuperFreezZ**](https://gitlab.com/SuperFreezZ/SuperFreezZ) <sup>**[[F-Droid](https://f-droid.org/app/superfreeze.tool.android)]**</sup>
 * [**Battery Tool**](https://github.com/Domi04151309/BatteryTool) <sup>**[[F-Droid](https://f-droid.org/app/io.github.domi04151309.batterytool)]**</sup>
 * [**AppManager**](https://github.com/MuntashirAkon/AppManager) <sup>**[[F-Droid](https://f-droid.org/app/io.github.muntashirakon.AppManager)]**</sup>
+* [**Inure App Manager**](https://github.com/Hamza417/Inure)
```
Comment (offa) [inline README.md:567]: According to [this discussion](https://github.com/Hamza417/Inure/issues/11) all proprietary parts are removed :+1:.

**Review by offa — APPROVED** [review]

**Comment by offa** [comment]
Thank you @shuvashish76! :+1: 

> Quillnote replaced with it's fork Quillpad (Is this project abandoned? msoultanidis/quillnote#177 (comment))

Nice to see someone continuing the project.

File: README.md (hunk 2 above)
Comment (shuvashish76) [reply README.md:567]: Change app name to "Inure App Manager (Trial)"
Also live on F-Droid: https://f-droid.org/en/packages/app.simple.inure/

Comment (offa) [reply README.md:567]: Updated, thanks!

I don't find much information what _Trial_ version actually does. What happens after the 15 days?

Comment (shuvashish76) [reply README.md:567]: IDK never tried it as most of the premium features of Inure App Manager are free in [AM](https://github.com/MuntashirAkon/AppManager) + active development.

Comment (shuvashish76) [reply README.md:567]: https://hamza417.gumroad.com/l/inure_unlocker/

**Pull request merged**